docs/usage/getting-started/private-packages.md

```diff
@@ -391,6 +391,25 @@ hostRules: [
 ];
 ```
 
+### pip
```

Comment:
Do we want to create a new section in the Private packages page, and have an example of overriding the `index-url` in your `requirements.txt` file?

We already have a page that sorta-kinda explains this:

https://docs.renovatebot.com/python/#specify-url-in-configuration